org.inkscape.Inkscape.json

```diff
@@ -0,0 +1,192 @@
+                }
+            ]
+        },
+        {
+            "name": "boehmgc",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://http.debian.net/debian/pool/main/libg/libgc/libgc_7.4.2.orig.tar.gz",
+                    "sha256": "09f21d38dd42bced14d6875f55804cb1b5e6bc486da7980187ecd2e768739c62"
+                }
+            ]
+        },
+        {
+            "name": "cairomm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/cairomm/1.12/cairomm-1.12.0.tar.xz",
+                    "sha256": "a54ada8394a86182525c0762e6f50db6b9212a2109280d13ec6a0b29bfd1afe6"
+                }
+            ]
+        },
+        {
+            "name": "glibmm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/glibmm/2.48/glibmm-2.48.1.tar.xz",
+                    "sha256": "dc225f7d2f466479766332483ea78f82dc349d59399d30c00de50e5073157cdf"
+                }
+            ]
+        },
+        {
+            "name": "pangomm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/pangomm/2.40/pangomm-2.40.1.tar.xz",
+                    "sha256": "9762ee2a2d5781be6797448d4dd2383ce14907159b30bc12bf6b08e7227be3af"
+                }
+            ]
+        },
+        {
+            "name": "atkmm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/atkmm/2.24/atkmm-2.24.2.tar.xz",
+                    "sha256": "ff95385759e2af23828d4056356f25376cfabc41e690ac1df055371537e458bd"
+                }
+            ]
+        },
+        {
+            "name": "gtkmm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/gtkmm/2.24/gtkmm-2.24.5.tar.xz",
+                    "sha256": "0680a53b7bf90b4e4bf444d1d89e6df41c777e0bacc96e9c09fc4dd2f5fe6b72"
+                }
+            ]
+        },
+        {
+            "name": "potrace",
+            "config-opts": ["--with-libpotrace"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://potrace.sourceforge.net/download/1.14/potrace-1.14.tar.gz",
+                    "sha256": "db72b65311cfdcb63880b317f610d84485f086e15f88ca2346012d49414cd97e"
+                }
+            ]
+        },
+        {
+            "name": "lxml",
+            "config-opts": ["--module=lxml==3.7.3"],
+            "build-options": {
+                "env": {
+                    "XSLT_CONFIG": "pkg-config libxslt"
+                },
+                "build-args": ["--share=network"]
```

Comment:
Looks like GNOME Builder builds it offline though. Take a look at its [manifest](https://github.com/chergert/gnome-builder/blob/a054e31dccb10219e3cddb2fab58123db6f7d457/org.gnome.Builder.json#L184).